app/models/user.rb

```diff
@@ -1,6 +1,8 @@
 class User < ActiveRecord::Base
   has_many :cards, dependent: :destroy
+  has_many :blocks, dependent: :destroy
   has_many :authentications, dependent: :destroy
+  belongs_to :current_block, class_name: 'Block'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.